client/components/LayoutEditor/layoutEditor.scss

```diff
@@ -14,7 +14,7 @@
 	.component-wrapper {
 		margin-bottom: 2em;
 		border: 1px solid #ccc;
-	}
+	} 
```

Comment:
Might as well revert this (I'm working on getting a Prettier-style linter set up for scss so we won't have to worry about this manually)

Reply:
not the spacing *shakes fist in air* lol. gotcha